cpu/rpx0xx/periph/uart.c

```diff
@@ -67,7 +67,7 @@ void _set_symbolrate(uart_t uart, uint32_t baud)
     dev->UARTIBRD.reg = baud_ibrd;
     dev->UARTFBRD.reg = baud_fbrd;
 
-    io_reg_atomic_set(&(dev->UARTLCR_H.reg), 0);
+    dev->UARTLCR_H.reg = 0;
```

Comment:
This shouldn't be needed, actually. (This also explains why this worked with the previous no-op atomic `|= 0` operation). Because:

1. `0x0` is the reset value of the register. So after the reset which is triggered during `uart_init()`, it will have this value anyway
2. In `uart_mode()` the register is overwritten anyway

I'll just drop this.